plugins/autopeering/plugin.go

```diff
@@ -256,29 +258,79 @@ func configureEvents() {
 		Plugin.LogInfof("removed offline: %s / %s", ev.Peer.Address(), peerID.ShortString())
 	})
 
+	onPeerConnected = events.NewClosure(func(p *p2p.Peer, conn network.Conn) {
+
+		if deps.AutopeeringManager.Selection() == nil {
+			return
+		}
+
+		id := autopeering.ConvertPeerIDToHiveIdentityOrLog(p, Plugin.LogWarnf)
+		if id == nil {
+			return
+		}
+
+		// we block peers that are connected via manual peering in the autopeering module.
+		// this ensures that no additional connections are established via autopeering.
+		switch p.Relation {
+		case p2p.PeerRelationKnown:
```

Comment:
you can combine both cases with a `,` if you are doing the exact same thing for both